maggy/core/environment/__init__.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from maggy.core.environment import abstractenvironment, baseenvironment
+
+AbstractEnvironment = abstractenvironment.AbstractEnvironment
+# check which environment to use
+if "REST_ENDPOINT" in os.environ:
+    print("You are running maggy on Hopsworks.")
+
+    from maggy.core.environment import hopsenvironment
+
+    HopsEnvironment = hopsenvironment.HopsEnvironment
+
+    __all__ = ["AbstractEnvironment", "HopsEnvironment"]
```

Comment:
I am not sure I understand why you need the check for the environment also here. This seems redundant. I usually use the `__all__` global only when I want to make a user facing API available with less typing. E.g. take a look at the `Searchspace` class, there I use it so the user can do `from maggy import Searchspace` instead of `from maggy.searchspace import Searchspace`. Since this is not a class the User should use himself anyway, I would get rid of all of this code in this file (except copyright header).